test/testSetup.ts

```diff
@@ -132,26 +133,9 @@ export class TestSetup {
     utilities.log('');
     if (!Env.getInstance().useExistingProject) {
       utilities.log(`${projectName ?? this.testSuiteSuffixName} - Starting createProject()...`);
-      this.prompt = await utilities.executeQuickPick('SFDX: Create Project');
```

Comment:
Please add a new test case for `SFDX: Create Project` maybe inside miscellaneous so we don't lose coverage for the command

Reply:
This will be addressed in follow up [WI](https://gus.lightning.force.com/lightning/r/a07EE000020G7FrYAK/view) as discussed on slack.